eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md

```diff
@@ -221,6 +189,24 @@ __Legal targets:__ [EiffelSourceChangeCreatedEvent](../eiffel-vocabulary/EiffelS
 __Multiple allowed:__ Yes  
 __Description:__ Identifies a latest previous version (there may be more than one in case of merges) of the created source change.
 
+### PARTIALLY_RESOLVED_ISSUE
+__Required:__ No  
+__Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDefinedEvent.md)  
+__Multiple allowed:__ Yes  
+__Description:__ Identifies an issue that this event partially resolves. That is, this SCC introduces some change that has advanced an issue towards a resolved state, but not completely resolved.
+
+### RESOLVED_ISSUE
+__Required:__ No  
+__Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDefinedEvent.md)  
+__Multiple allowed:__ Yes  
+__Description:__ Identifies an issue that this SCC is claiming it has done enough to resolve. This is not an authoritative resolution, only a claim.
```

Comment:
Probably worth elaborating on this, e.g. "The issue itself may or may not change status as a consequence of this, e.g. following [successful verification](../eiffel-vocabulary/EiffelIssueVerified.md) of the issue or due to manual update."